autograph.yaml

```diff
@@ -90,6 +90,7 @@ signers:
         numgenerators: 2
         generatorsleepduration: 1m
         fetchtimeout: 100ms
+        monitorsamplerate: 1s
```

Comment:
isn't 1s a bit aggressive?

Reply:
Yeah that was just to see it was working locally. Bumped to 1 min.

@milescrabill any idea what we'll want to use in stage and prod?